.github/workflows/CI.yml

```diff
@@ -1075,5 +1131,8 @@ jobs:
         run: |
           ssm_pkg_version=`cat build/packages/VERSION`
           aws ssm describe-document --name ${SSM_PACKAGE_NAME} --version-name ${ssm_pkg_version} >/dev/null 2>&1 && \
-            aws ssm delete-document --name ${SSM_PACKAGE_NAME} --version-name ${ssm_pkg_version}
+            for region in $INTEGRATION_TEST_REGION_1 $INTEGRATION_TEST_REGION_2
```

Comment:
are we planning on adding more test regions? because if its only two it may be simpler to just write the 2 commands

Reply:
Yes. I assume we would add more ask our test growing. For now, I'm fine with YAGNI.

Reply:
Might be better to use a matrix strategy.
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
Reasonable. Fixed it for the newest commit. For ```ec2-preparation``` job, I thought of using it too but it ways too overkill for this PR. Need to separate the release S3 bucket - SSM package name and  build-image. Added matrix in the CD too and for resources weekly cleaning.